.github/workflows/format.yml

```diff
@@ -17,22 +17,22 @@ jobs:
           fetch-depth: 1000
 
       - name: Configure git
-        run: |
-          git config --global user.email "-"
-          git config --global user.name "Autoformatter"
-          git fetch origin "$GITHUB_BASE_REF":"$GITHUB_BASE_REF" --depth=1000
+        run: git fetch origin "$GITHUB_BASE_REF":"$GITHUB_BASE_REF" --depth=1000
 
       - name: Install clang-format
         run: |
           sudo apt-get update
           sudo apt-get -y install clang-format-12
 
       - name: Check formatting
-        run: tests/test-format.sh
+        run: |
+          git clang-format-12 --extensions cpp,h --style file "$GITHUB_BASE_REF"
```

Comment:
On my system this would be `git-clang-format` with the space replaced by a hyphen. Do you know if either one's standard, and if the other one's something Ubuntu adds for example?

Reply:
As far as I know, as long as tge git-clang-format executable is in you PATH, it can be called vie git clang-format. See the comment at the top of this file: https://github.com/llvm/llvm-project/blob/main/clang/tools/clang-format/git-clang-format.

Reply:
I see. I must've typo'd earlier when I tried it.